Resources/Private/BackendModule/src/Components/AssetListItem.tsx

```diff
@@ -128,16 +107,16 @@ export default class AssetListItem extends PureComponent {
                             style={{marginRight: '8px'}}
                             disabled={!this.canDiscard()}
                             onClick={this.discard.bind(this)}>
```

Comment:
Kleinigkeit, statt der Bind-Methode würd ich immer eine fat arrow function nutzen, also:
```onClick={() => this.discard()}>```

Reply:
Ist nicht dasselbe. Wenn ich das so schreibe, passiert nichts.

Reply:
Okay, nehme ich zurück. Es hat jetzt doch geklappt. Wenn man halt das Event benötigt, muss man es so schreiben: `(e) => this.handleChange(e)`